modules/connatixBidAdapter.js

```diff
@@ -308,6 +333,22 @@ export const spec = {
       params['us_privacy'] = encodeURIComponent(uspConsent);
     }
 
+    window.addEventListener('message', function handler(event) {
+      if (!event.data || event.origin != 'https://cds.connatix.com') {
+        return;
+      }
+
+      if (event.data.type === 'cnx_all_identity_providers_resolved') {
```

Comment:
We shouldn't just listen for the final event, we should listen to every update event and update our local storage accordingly

Reply:
im listening for all events, except when i get all providers resolved event, i stop listening, as i assumed that once i get this one there are no more events coming

Reply:
I'd listen for each event specifically, if resolvedProvider do something, if allResolved fo something. If in the future we add other events that don't update ids you'll get errors here

Reply:
i understand, i modified it now

Reply:
👍